resources/application.rb

```diff
@@ -37,7 +37,7 @@
 attribute :cron_restart,        :kind_of => String
 attribute :cwd,                 :kind_of => String, :required => true
 attribute :merge_logs,          :kind_of => [TrueClass, FalseClass]
-attribute :watch,               :kind_of => [TrueClass, FalseClass]
```

Comment:
Would you mind adding `String` as well?

Based on the docs at: http://pm2.keymetrics.io/docs/usage/watch-and-restart/